src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/OpenAppManifestValueProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
```

Comment:
nit: For new code files, we've started using file-scoped namespaces to reduce indentation. Feel free to use that for these new classes here. If you type a `;` at the end of this line in VS, it'll convert automatically for you.